src/applications/vaos/referral-appointments/ScheduleReferral.unit.spec.js

```diff
@@ -1,35 +1,36 @@
 import React from 'react';
 import { expect } from 'chai';
-import sinon from 'sinon';
-import ScheduleReferral from './ScheduleReferral';
 import {
   createTestStore,
   renderWithStoreAndRouter,
 } from '../tests/mocks/setup';
-import { referral } from './temp-data/referral';
+import ScheduleReferral from './ScheduleReferral';
+import { createReferral } from './utils/referrals';
 
 describe('scheduleReferral component', () => {
   it('should display the subtitle correctly given different numbers of appointments', async () => {
+    const referralOne = createReferral(new Date(), '111');
     const store = createTestStore();
-    const sandbox = sinon.createSandbox();
-    sandbox.stub(referral, 'appointmentCount').value(1);
-    const screen = renderWithStoreAndRouter(<ScheduleReferral />, {
-      store,
-    });
+    const screen = renderWithStoreAndRouter(
+      <ScheduleReferral currentReferral={referralOne} />,
+      {
+        store,
+      },
+    );
     const subtitle = await screen.findByTestId('subtitle');
     expect(subtitle).to.contain.text('1 appointment');
-    sandbox.restore();
   });
   it('should display the subtitle correctly given 2 appointments', async () => {
+    const referralTwo = createReferral(new Date(), '222');
     const store = createTestStore();
-    const sandbox = sinon.createSandbox();
-    sandbox.stub(referral, 'appointmentCount').value(2);
-    referral.appointmentCount = 2;
-    const screen = renderWithStoreAndRouter(<ScheduleReferral />, {
-      store,
-    });
+    referralTwo.numberOfAppointments = 2;
+    const screen = renderWithStoreAndRouter(
```

Comment:
I think we can just use `render` now that we are passing in the referral

Reply:
we need the store and router stuff because the page uses the `useLocation` hook